tests/components/combined_energy/test_config_flow.py

```diff
@@ -0,0 +1,166 @@
+"""Test cases for combined energy config flow."""
+from unittest.mock import patch
+
+import combined_energy.exceptions
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.combined_energy.const import CONF_INSTALLATION_ID, DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+
+
+async def test_form(hass: HomeAssistant) -> None:
+    """Test that a form is created."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+
+    assert result["type"] == "form"
```

Comment:
`assert result["type"] == FlowResultType.FORM` seems better.
(happens multiple times in this file)